packages/client/components/TeamPrompt/TeamPromptRepliesAvatarList.tsx

```diff
@@ -0,0 +1,103 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import useOverflowAvatars from '~/hooks/useOverflowAvatars'
+import {TransitionStatus} from '~/hooks/useTransition'
+import {TeamPromptRepliesAvatarList_edges$key} from '~/__generated__/TeamPromptRepliesAvatarList_edges.graphql'
+import AvatarListUser from '../AvatarListUser'
+import TeamPromptOverflowAvatar from './TeamPromptOverflowAvatar'
+
+const Wrapper = styled('div')<{minHeight: number; width: number}>(({minHeight, width}) => ({
+  position: 'relative',
+  minHeight,
+  width,
+  marginRight: '8px'
+}))
+
+interface Props {
+  edgesRef: TeamPromptRepliesAvatarList_edges$key
+}
+
+const SIZE = 24
+const OVERLAP = 4
```

Comment:
+1 could be more specific: What size? Which overlap?

Reply:
Prefixed each with `AVATAR_`